plugins/Ecoji/plugin.json

```
@@ -0,0 +1,6 @@
+{
+  "name": "Ecoji",
+  "version": "0.1.1",
+  "authors": "Mukto"
+  "main_class": "Ecoji"
+}
```

```suggestion
{
  "name": "Ecoji",
  "version": "0.1.0",
  "authors": "Mukto",
  "main_class": "Ecoji"
}
```

starting version should be 0.1.0 instead of 0.1.1, also there was a missing `,`